fractions/jaeger/src/main/resources/modules/com/uber/jaeger/main/module.xml

```diff
@@ -0,0 +1,18 @@
+<module xmlns="urn:jboss:module:1.3" name="com.uber.jaeger">
+  <resources>
+    <artifact name="com.uber.jaeger:jaeger-core:${version.jaeger}"/>
+    <artifact name="com.uber.jaeger:jaeger-tracerresolver:${version.jaeger}"/>
+    <artifact name="com.uber.jaeger:jaeger-thrift:${version.jaeger}"/>
+    <artifact name="org.apache.thrift:libthrift:0.9.2"/>
```

Comment:
Transitive dependencies from Maven are *not* considered by JBoss-Modules and the `module.xml` files.  You do have to be explicit about defining module.xml's and what artifacts go into them.